metaflow/parameters.py

```diff
@@ -227,7 +227,7 @@ def add_custom_parameters(deploy_mode=False):
     # deploy_mode determines whether deploy-time functions should or should
     # not be evaluated for this command
     def wrapper(cmd):
-        for arg in parameters:
+        for arg in parameters[::-1]:
```

Comment:
Thanks for making the change! Can we just add a quick comment above this line so that a future refactor can preserve this behavior?

Reply:
No prob. Done!

Reply:
Thank you!